integration_test/migration_test.go

```diff
@@ -42,6 +60,27 @@ func TestKrewUnsupportedVersion(t *testing.T) {
 	}
 }
 
+func isIndexMigrated(it *ITest) bool {
+	indexPath := it.TempDir().Path("index/default")
+	_, err := os.Stat(indexPath)
+	return err == nil
+}
+
+func prepareOldIndexLayout(it *ITest) {
```

Comment:
is this something we should eventually delete?
if so please add a TODO (e.g. // TODO remove when testing indexmigration is no longer necessary).